iotedged/src/main.rs

```diff
@@ -378,6 +381,18 @@ struct Options {
 	#[structopt(long)]
 	device_id: String,
 
+	/// `keyd` API endpoint
+	#[structopt(long, default_value = "unix:///var/lib/aziot/keyd.sock")]
+	keyd_url: url::Url,
```

Comment:
Update `Cargo.toml` to enable the `"serde"` feature on the `url` dependency. It'll compile fine already because one of the other deps enables it, but it's good to be explicit.

Reply:
IIRC, structopt uses `FromStr` to parse custom data structures, not serde, so I don't think this should be necessary

Reply:
Oh right, derp.